1.0/docs/devguide/templates.md

```diff
@@ -303,6 +303,30 @@ If you or a third-party library mutate the array without Polymer's methods,
 you need to call [`notifySplices`](#notifysplices) to ensure that any elements
 watching the array are properly notified.
 
+### Performance properties {#dom-repeat-perf}
+
+Use the following properties to affect the performance of `dom-repeat` 
+templates. 
+
+* [`delay`](/1.0/api/#dom-repeat:property-delay). Set a debounce time 
```

Comment:
Make sure these links work correctly. I think we had to do something wacky before like include the full URL.